app/javascript/app/document-capture/context/i18n.js

```diff
@@ -0,0 +1,5 @@
+import { createContext } from 'react';
+
+const I18nContext = createContext({});
```

Comment:
react noob question: what is the benefit of creating an empty context like this in? Is this just a shared object to persist translations in?

Reply:
The argument represents the default value to be used by any consumers of the context in the absence of any provider. Essentially, we're saying that the value of `context` will always be an object, where in most cases we'd assume that the object would be overridden with the locale data of the application.

While React won't object to it, most agree that a default should always be provided. The closest thing to the "official" TypeScript React types won't allow for it to be omitted;

- https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e9a0acff54a30d5e5454690de87317bcf51289ef/types/react/index.d.ts#L382-L386
- https://github.com/DefinitelyTyped/DefinitelyTyped/pull/24509#issuecomment-382213106

But more importantly, I take advantage of this in the tests, where [`<DocumentCapture />` is rendered without the context being provided](https://github.com/18F/identity-idp/pull/3909/files#diff-40fe6c78a746f546435f964350a405fbR10), and it still "works"†. 

_† With the caveat as noted in the original comment about how this falls back to rendering the text of the translation key in absence of real locale data._